src/posts/action.md

```diff
@@ -3,56 +3,56 @@ title: Take action
 description: Ways to help out with pausing AGI development.
 ---
 
-The group of people who are aware of AI risks is still small.
+AI will won't get safer unless we act decisively to push for safety. Choose an activity below from your interest / skill areas and help save the future!
 
-You are now one of them.
+## For everyone
 
-**So your actions matter more than you think.**
+### 1. Encourage governments to act
 
-Here are some examples of what you can do.
+- [**Create an email for a politician**](/email-builder) **in the** [**US**](https://democracy.io/): Emails are completely overpowered. Try to convince your government to work towards a pause or enforce other binding AI safety legislation. Here are some [**lobby tips**](/lobby-tips).
```

Comment:
Why the US focus? This could dissuade people from writing if they are in non-US countries IMO.

Reply:
Yeah, just Link the US stuff (if it's not already linked) on the US lobby page.
Also "Write" or "Send" sounds better than "Create"